### PR TITLE
Add note about certs on CentOS

### PR DESCRIPTION
**Pull request by JimGalasyn**
Inserted admonition for for adding certs in CentOS 6 & 7, and in CentOS 5.

### CODE REVIEW

File: notary/running_a_service.md [hunk 1]
```diff
@@ -200,6 +200,37 @@ environment.
 The sample private key files in the Notary repository are obviously public knowledge
 and using them in a production deployment is highly insecure.
 
+> Certificates on CentOS
```
Comment (johndmulhausen) [inline notary/running_a_service.md]: This should be bolded, yeah?

File: notary/running_a_service.md [hunk 2]
```diff
@@ -200,6 +200,37 @@ environment.
...
+>     ```bash
+>     $ update-ca-trust extract
+>     ```
+>
+> For CentOS 5, just add your certificate to the bundle.
+> -   Append your trusted certificate to the bundle file at
+>     `/etc/pki/tls/certs/ca-bundle.crt`:
+>     ```bash
+>     $ cat your-cert.crt >> /etc/pki/tls/certs/ca-bundle.crt
+>     ```
```
Comment (johndmulhausen) [inline notary/running_a_service.md]: ^ Is this all rendering okay? Potential issues I see:

1. No line break between bullet item text and code snippets
2. Bullet list + blockquote + code snippet indentation = dicey at times
3. Not sure if/why "Append your trusted certificate" is a bullet item, unclear how the indentation + backtick combination will work on the text below.

I wish Netlify were working...

Comment (JimGalasyn) [inline notary/running_a_service.md]: K I took out the fancy formatting, to be safe.

**Comment by johndmulhausen** [comment]
Beautiful, thanks @JimGalasyn :)

**Pull request merged**